kedro_vertexai/cli.py

```diff
@@ -101,16 +110,65 @@ def run_once(
     context_helper = ctx.obj["context_helper"]
     config: RunConfig = context_helper.config.run_config
     client: VertexAIPipelinesClient = context_helper.vertexai_client
+    image: str = image if image else config.image
+
+    reminder = not config.no_reminder
+    if auto_build:
+        try:
+            # Subprocess docker build
+            proc = subprocess.Popen(
```

Comment:
`subprocess.Popen` is very low-level API, consider using [subprocess.run(..., check=True)](https://docs.python.org/3/library/subprocess.html#subprocess.run)

Reply:
https://docs.python.org/3/library/subprocess.html#subprocess.check_output ;)

Reply:
done, not using check, doing it manually